typedb-src/modules/ROOT/pages/admin/configuration.adoc

```diff
@@ -157,7 +157,14 @@ The log section of the configuration file contains the logging behavior options
 
 There are three subsections:
 
-* `output`: Define destinations to write logs to. Allowed types are `type: file`, and `type: stdout` in TypeDB.
+* `output`: Define destinations to write logs to.
+** User-defined output channel name
+*** `type` -- it's either `file` or `stdout`.
+*** `directory` -- filepath, relative to the server binary. Only available for `type: file`.
+*** `file-size-cap` -- maximum size of a log file. If the log file reaches the limit, we start writing in a new file
+    in the same directory. This is similar to the `maxsize` config option in logrotate. Only available for `type: file`.
+*** `archives-size-cap` -- maximum size of all log files. If the summ of all log files in the directory reaches
+    the limit, we remove the oldest one. Only available for `type: file`.
```

Comment:
Same comment for 'we' here

Reply:
Addressed the issue in https://github.com/vaticle/typedb-docs/pull/688/commits/3acbc02d2bb430adf1dc48b2c616a0d8d773c87f